src/snowcli/output/printing.py

```diff
@@ -23,6 +23,28 @@ def default(self, o):
         return super().default(o)
 
 
+def print_db_cursors(cursors: List[SnowflakeCursor]) -> None:
+    """
+    Prints results fetched by cursor using specified format.
+
+    :param cursors: list of snowflake cursors for fetching results
+    :param columns: list of columns that should be included in output, if
+        not provided then all columns are returned
+    :return:
+    """
+    output_format = _get_format_type()
+    if output_format == OutputFormat.TABLE:
+        for cursor in cursors:
+            print_db_cursor(cursor)
+    elif output_format == OutputFormat.JSON:
+        import json
+
+        data = [_get_data_from_cursor(c) for c in cursors]
+        print_json(json.dumps(data, cls=CustomJSONEncoder))
+    else:
+        raise Exception(f"Unknown {output_format} format option")
```

Comment:
Maybe someone will have better idea with this format parameter check. I do it twice for table output :/

Reply:
First thing that comes to my mind is keyword argument, with default value set to None, both in `print_db_cursor` and `print_formatted` - so in the latter, it would be `output_format= output_format if output_format else _get_output_format`